### PR TITLE
Database connection

### LINKED ISSUE #9: Set up Database
## Value statement - a requirement, not a solution
As a user
I need my data to be saved on a database
so that i can access it from anywhere without being stuck on a local device.

## Description 

Implement a backend! The data should be saved on a database, instead of localStorage.

## Acceptance criteria
- [ ] every data in communiFI is saved in a MongoDB
- [ ] every data in communiFI is read out of a MongoDB
- [ ] the data is seperated into multiple clusters (one cluster for expenses, one for revenues and so on)

## Tasks
- [ ] create a Cluster in MongoDB
- [ ] create multiple collections in MongoDB
- [ ] create schema with mongoose for the different collections
- [ ] write a connectodb function in your project
- [ ] dock mongo to the services and api endpoints 

## Size
medium

### PR DESCRIPTION
**Pull request by edanum**
This PR fixes #9 

It includes a simple Mongo DB from which the data of the expense page is generated.
FYI: The User-Story criteria isnt met completly because I didnt implement the add and edit function yet.

### CODE REVIEW

File: models/Expense.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Decimal128, Int32 } from "bson";
```
Comment (Kheirah) [inline models/Expense.js]: Are you going to need this in the future? It's not being used at the moment from what I can tell.

File: models/Expense.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const { Schema } = mongoose;
+
+const expenseSchema = new Schema({
+  receiptNumber: { type: Number, rquired: true },
+  amount: { type: Number, rquired: true },
+  description: { type: String, rquired: true },
+  dateOfSubmit: { type: String, rquired: true },
+  comment: { type: String, rquired: false },
+  receipt: { type: String, rquired: false },
+  name: { type: String, rquired: true },
```
Comment (Kheirah) [inline models/Expense.js]: Typo =)

You meant to say "required" instead of "rquired".

**Review by Kheirah — APPROVED** [review]
The DB connection for GET requests responds as expected. Good work!

There's just a small typo in the schema, easily fixed and you're good to go!

File: models/Expense.js (hunk 1 above)
Comment (edanum) [reply models/Expense.js]: Nope, I tried to use it but it didnt work so i sticked with "number" and forgot to delete the import. Thanks

File: models/Expense.js (hunk 2 above)
Comment (edanum) [reply models/Expense.js]: haha, damn. So obvious. Thanks

**Pull request merged**